src/test/scenarios/datafactory/output/src/datafactory/azext_datafactory/generated/custom.py

```diff
@@ -123,3 +125,291 @@ def datafactory_get_git_hub_access_token(cmd, client,
+
+
+def datafactory_integration_runtime_list_auth_key(cmd, client,
+                                                  resource_group_name,
+                                                  factory_name,
+                                                  integration_runtime_name):
+    return client.list_auth_key(resource_group_name=resource_group_name,
+                                factory_name=factory_name,
+                                integration_runtime_name=integration_runtime_name)
+
+
+def datafactory_integration_runtime_regenerate_auth_key(cmd, client,
+                                                        resource_group_name,
+                                                        factory_name,
+                                                        integration_runtime_name,
+                                                        regenerate_key_parameters):
+    if isinstance(regenerate_key_parameters, str):
+        regenerate_key_parameters = json.loads(regenerate_key_parameters)
+    return client.regenerate_auth_key(resource_group_name=resource_group_name,
+                                      factory_name=factory_name,
+                                      integration_runtime_name=integration_runtime_name,
+                                      regenerate_key_parameters=regenerate_key_parameters)
+
+
+def datafactory_integration_runtime_remove_link(cmd, client,
+                                                resource_group_name,
+                                                factory_name,
+                                                integration_runtime_name,
+                                                linked_factory_name):
+    return client.remove_link(resource_group_name=resource_group_name,
+                              factory_name=factory_name,
+                              integration_runtime_name=integration_runtime_name,
+                              linked_factory_name=linked_factory_name)
+
+
+def datafactory_integration_runtime_start(cmd, client,
+                                          resource_group_name,
+                                          factory_name,
+                                          integration_runtime_name,
+                                          no_wait=False):
+    return sdk_no_wait(no_wait, client.begin_start,
+                                                  resource_group_name=resource_group_name,
+                                                  factory_name=factory_name,
+                                                  integration_runtime_name=integration_runtime_name)
+
+
+def datafactory_integration_runtime_stop(cmd, client,
+                                         resource_group_name,
+                                         factory_name,
+                                         integration_runtime_name,
+                                         no_wait=False):
+    return sdk_no_wait(no_wait, client.begin_stop,

```

Comment:
is it possible to have better format? :)

Reply:
what kind of format does CLI need?

Reply:
Changed, please review again.